src/main/java/uk/gov/hmcts/sptribs/common/event/CreateTestCase.java

```diff
@@ -46,6 +46,12 @@ public class CreateTestCase implements CCDConfig<CaseData, State, UserRole> {
     @Autowired
     private CcdPageConfiguration selectParties;
 
+    /*@Autowired
+    private CcdPageConfiguration applicantDetails;
+
+    @Autowired
+    private CcdPageConfiguration caseCategorisationDetails;*/
+
```

Comment:
Do we still need this commented code?

Reply:
I left it by accident. will remove thanks